config/application.rb

```diff
@@ -220,6 +220,11 @@ class Application < Rails::Application
     # Active job should use resque
     config.active_job.queue_adapter = :resque
 
+    # Show the GDPR "I accept" cookies banner by default.
+    # This setting is overridden in test.rb,
+    # so the tests don't have to click "I accept".)
+    config.hide_accept_cookies_banner = true
```

Comment:
This is a bug -- it should actually be false. We do not want to hide the cookie banner by default, we only want to hide it for the unit tests. Fixed in https://github.com/sciencehistory/chf-sufia/pull/1155 .